src/src_main/crypto_tc.c

```diff
@@ -213,8 +220,9 @@ int32_t Crypto_TC_ApplySecurity(const uint8_t *p_in_frame, const uint16_t in_fra
         case SA_AUTHENTICATION:
             // Ingest length + spi_index (2) + shivf_len (varies) + shsnf_len (varies)
             //   + shplf_len + arc_len + pad_size + stmacf_len
+            // TODO: If ARC is transmitted in the SHSNF field (as in CMAC... don't double count those bytes)
```

Comment:
Should make sure we create an issue for the couple of TODOs you've already identified